src/dmd/root/outbuffer.d

```diff
@@ -459,4 +459,3 @@ char[] unsignedToTempString(ulong value, char[] buf, uint radix = 10) @safe
     } while (value);
     return buf[i .. $];
 }
-
```

Comment:
Seems like a leftover from a previous attempt here.

Reply:
It is, but also its own commit